test/unit/object/test_item.py

```diff
@@ -320,15 +330,25 @@ def test_collaborate_with_user(test_item_and_response, mock_user, mock_box_sessi
             'id': '1111',
         }
     }
+    expected_params = params
     mock_box_session.post.return_value.json.return_value = mock_collaboration
-    collaboration = test_item.collaborate(mock_user, 'editor')
-    mock_box_session.post.assert_called_once_with(expected_url, data=json.dumps(expected_data), params={})
+    collaboration = test_item.collaborate(mock_user, 'editor', can_view_path=can_view_path, fields=fields, notify=notify)
+    mock_box_session.post.assert_called_once_with(expected_url, data=json.dumps(expected_data), params=expected_params)
     assert collaboration.id == mock_collaboration['id']
     assert collaboration['type'] == mock_collaboration['type']
     assert collaboration['created_by']['id'] == mock_collaboration['created_by']['id']
 
 
-def test_collaborate_with_login(test_item_and_response, mock_box_session):
+@pytest.mark.parametrize('can_view_path,fields,notify,data,params', [
+    (None, None, None, {}, {}),
+    (True, None, None, {'can_view_path': True}, {}),
+    (False, None, None, {'can_view_path': False}, {}),
+    (None, ['type', 'id', 'created_by'], None, {}, {'fields': 'type,id,created_by'}),
+    (None, None, True, {}, {'notify': True}),
+    (None, None, False, {}, {'notify': False}),
+    (True, ['type', 'id', 'created_by'], False, {'can_view_path': True}, {'fields': 'type,id,created_by', 'notify': False})
+])
```

Comment:
Is this the exact same params as for `test_collaborate_with_user`? May want to DRY this up.